framework/physics/SolverBase/chi_solver.cc

```diff
@@ -176,4 +180,37 @@ Solver::GetInfoWithPreCheck(const chi::ParameterBlock& params) const
   return GetInfo(params);
 }
 
+/**\addtogroup SolverBase
+*
+* \section Properties Properties that can be set
+* The following properties can be set via the lua call
+* `chi_lua::chiSolverSetProperties`
+* \copydoc chi_physics::Solver::SetProperties*/
+
+/**
+Base solver settable properties:
+* - `dt`, Timestep size
+* - `time`, Current time
+* */
+void Solver::SetProperties(const chi::ParameterBlock& params)
+{
+  for (const auto& param : params)
+  {
+    const std::string param_name = param.Name();
+
+    if (param_name == "dt")
+      timestepper_->SetTimeStepSize(param.GetValue<double>());
+    if (param_name == "time")
+      timestepper_->SetTime(param.GetValue<double>());
+    if (param_name == "start_time")
+      timestepper_->SetStartTime(param.GetValue<double>());
+    if (param_name == "end_time")
+      timestepper_->SetEndTime(param.GetValue<double>());
+    if (param_name == "max_time_steps")
+      timestepper_->SetMaxTimeSteps(param.GetValue<int>());
+    if (param_name == "dt_min")
+      timestepper_->SetMinimumTimeStepSize(param.GetValue<int>());
+  }
```

Comment:
The above is a pretty nice paradigm. Notice the changes in the PRK solver just for me to see if we can link the documentation together.